### PR TITLE
feat(SlidePanel/TSCH-52): new toggle button

### PR DESCRIPTION
**Pull request by vyshkov**
**What is the problem this PR is trying to solve?**

Frontify definition for SlidePanel updated http://guidelines.talend.com/document/92132#/navigation-layout/left-side-panel.

In this PR i changed only the button. Hover color will be updated with all colors in separate PR.

**What is the chosen solution to this problem?**

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

### CODE REVIEW

File: packages/components/stories/SidePanel.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ const icons = {
 	'talend-dataprep': talendIcons['talend-dataprep'],
 	'talend-download': talendIcons['talend-download'],
 	'talend-star': talendIcons['talend-star'],
+	'talend-opener': talendIcons['talend-opener'],
```
Comment (talend-sromain) [inline packages/components/stories/SidePanel.js]: Can you also add this icon to containers stories ( like homelist view )

File: packages/components/src/SidePanel/SidePanel.scss [hunk 2]
```diff
@@ -6,6 +6,8 @@
 /// Side panel toggle button color
 /// @type Color
 $tc-side-panel-toggle-color: tint($brand-primary, 75);
+$list-item-hover-color: darken($brand-primary, 2%);
```
Comment (talend-sromain) [inline packages/components/src/SidePanel/SidePanel.scss]: It's not the background color of the guideline ( maybe in another pr ? )
Also, the selected background color is not the same in the guideline
Would we do another pr for both background color & this one is only about toggle button ?

Comment (vyshkov) [reply packages/components/src/SidePanel/SidePanel.scss]: @talend-sromain i'll update all sidebar colors in separate PR. Because if i update only toggle button (to use white highlight) it will look bad.

Comment (mloussot) [reply packages/components/src/SidePanel/SidePanel.scss]: @vyshkov indeed, it would... can you add me as reviewer also on this other PR ? Thanks

Comment (vyshkov) [reply packages/components/src/SidePanel/SidePanel.scss]: @mloussot, ok, i'll add you

Comment (mloussot) [reply packages/components/src/SidePanel/SidePanel.scss]: Thanks
